doc/conf.py

```diff
@@ -13,10 +13,14 @@
 import os
 import sys
 
-from pyflagser import __version__
-
+# This line should be called before importing pyflagser
+# Otherwise it would be necessary to install the package in order to compile
+# the documentation
 sys.path.insert(0, os.path.abspath('../'))
 
+from pyflagser import __version__ # noqa
```

Comment:
I'm curious, wouldn't `from ..pyflagser import __version__` work?  (without needing the `sys.path.insert` above)

Reply:
Not 100% sure, but let me try

Reply:
I get the following exception:

`KeyError: "'__name__' not in globals`

Looking around, the [first](https://stackoverflow.com/questions/58938573/keyerror-name-not-in-globals-occurs-on-import-from-local-directory) result of google.
I think it is related to `__init__.py` files, I don't know if I want to spend time right now on that looking for a fix.
Do you prefer to have the relative import ?

Reply:
I guess I still wonder why we are not doing things the same way between `giotto-ph` and `pyflagser`.  Sorry, I am a little lost when it comes to docs.

Reply:
In `giotto-ph` we hard-code the value in `conf.py` which I think is not a good idea to duplicate the value on multiple files.
In `giotto-tda` we must have installed `giotto-tda` in order to build documentation, which I don't really like as an approach.

Interesting, it seems that in all 3 repositories, we do something similar as `sys.path.insert(0, os.path.abspath('../'))` in the `conf.py` file.